src/ontology/mondo-edit.obo

```diff
@@ -464918,6 +464920,2857 @@ is_a: MONDO:0005550 {source="https://orcid.org/0000-0002-5002-8648"} ! infectiou
+intersection_of: MONDO:0005583 {source="OMIA:002293"} ! non-human animal disease
+intersection_of: MONDO:0700097 MONDO:0009861 {source="OMIA:002293"} ! cross-species analog phenylketonuria
+
+[Term]
+id: MONDO:1010115
+name: c8 deficiency, non-human animal
+def: "C8 deficiency that occurs in non-human animals." [MONDO:patterns/nonhuman_disease]
+xref: OMIA:000156 {source="MONDO:equivalentTo"}
+is_a: MONDO:0005583 {source="OMIA:000156"} ! non-human animal disease
+intersection_of: MONDO:0005583 {source="OMIA:000156"} ! non-human animal disease
+intersection_of: MONDO:0700097 MONDO:0013422 {source="OMIA:000156"} ! cross-species analog type I complement component 8 deficiency
+
+[Term]
+id: MONDO:1010116
+name: leukocyte adhesion deficiency type I, non-human animal
+def: "Leukocyte adhesion deficiency type I that occurs in non-human animals." [MONDO:patterns/nonhuman_disease]
+xref: OMIA:000595 {source="MONDO:equivalentTo"}
+is_a: MONDO:0005583 {source="OMIA:000595"} ! non-human animal disease
+intersection_of: MONDO:0005583 {source="OMIA:000595"} ! non-human animal disease
+intersection_of: MONDO:0700097 MONDO:0007293 {source="OMIA:000595"} ! cross-species analog leukocyte adhesion deficiency 1
+
+[Term]
+id: MONDO:1010117
+name: severe combined immunodeficiency disease X-linked, non-human animal
+def: "Severe combined immunodeficiency disease X-linked that occurs in non-human animals." [MONDO:patterns/nonhuman_disease]
+xref: OMIA:000899 {source="MONDO:equivalentTo"}
+is_a: MONDO:0005583 {source="OMIA:000899"} ! non-human animal disease
+intersection_of: MONDO:0005583 {source="OMIA:000899"} ! non-human animal disease
+intersection_of: MONDO:0700097 MONDO:0010730 {source="OMIA:000899"} ! cross-species analog combined immunodeficiency, X-linked
+
+[Term]
+id: MONDO:1010118
+name: systemic lupus erythematosus, non-human animal
+def: "Systemic lupus erythematosus that occurs in non-human animals." [MONDO:patterns/nonhuman_disease]
+xref: OMIA:000968 {source="MONDO:equivalentTo"}
+is_a: MONDO:0005583 {source="OMIA:000968"} ! non-human animal disease
+intersection_of: MONDO:0005583 {source="https://orcid.org/0000-0002-5002-8648"} ! non-human animal disease
+intersection_of: MONDO:0700097 MONDO:0007915 {source="https://orcid.org/0000-0002-5002-8648"} ! cross-species analog systemic lupus erythematosus
+
+[Term]
+id: MONDO:1010119
+name: scleroderma, non-human animal
+def: "Scleroderma that occurs in non-human animals." [MONDO:patterns/nonhuman_disease]
+xref: OMIA:001202 {source="MONDO:equivalentTo"}
+is_a: MONDO:0005583 {source="OMIA:001202"} ! non-human animal disease
+intersection_of: MONDO:0005583 {source="https://orcid.org/0000-0002-5002-8648"} ! non-human animal disease
+intersection_of: MONDO:0700097 MONDO:0019340 {source="https://orcid.org/0000-0002-5002-8648"} ! cross-species analog scleroderma
+
+[Term]
+id: MONDO:1010120
+name: leukocyte adhesion deficiency type III, non-human animal
```

Comment:
In human, "disease ~ type x" is based on the gene affected. Should we have only the species-specific term(s) instead of this generic term based on "general species unspecific" gene? 
Note that these species-specific diseases (based on species-specific genes) could be analogous to the human type x disease, but it is difficult to create an axiom referring to the general "species-not- specific" gene.